src/fragments/start/getting-started/ionic/setup.mdx

```diff
@@ -59,8 +51,22 @@ When you initialize a new Amplify project, a few things happen:
 
 ## Install Amplify libraries
 
-In addition to `aws-amplify` core, install the Angular Ionic modules which provide a service provider, helpers, and components.
+Make sure to install the latest version of the `aws-amplify` core and the `@aws-amplify/ui-angular` library.
 
 ```sh
-$ npm install aws-amplify @aws-amplify/ui-angular@1.x.x
+$ npm install aws-amplify @aws-amplify/ui-angular
 ```
+
+The `@aws-amplify/ui-angular` package is a set of Angular components and an Angular provider which helps integrate your application with the `aws-amplify` library.
+
+<Callout warning={true}>
+  If you run into any errors after installing the ui-angular package, including
+  any issues with unhandled promise rejections or TypeScript errors, please see
+  our{' '}
+  <a
+    href="https://ui.docs.amplify.aws/angular/getting-started/installation#troubleshooting"
+    target="_blank"
```

Comment:
With @hbuchel's work on https://github.com/aws-amplify/amplify-ui/pull/1959, we might want to wait for that to merge then update this link.

Reply:
👍